packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -400,11 +419,17 @@ describe("SharedTree benchmarks", () => {
 							assert.equal(state.iterationsPerBatch, 1);
 							const provider = new TestTreeProviderLite(peerCount, factory);
 
-							// This is the start of the stream of commits.
-							// Earlier commits are less out of date and therefore not representative.
-							for (let iCommit = 0; iCommit < commitCount; iCommit++) {
-								for (let iPeer = 0; iPeer < peerCount; iPeer++) {
-									const peer = provider.trees[iPeer];
+							for (let iPeer = 0; iPeer < peerCount; iPeer++) {
+								const peer = provider.trees[iPeer];
+								updateSchema(peer, schemaGeneralized);
+								// Can't use assertSchema() because it sets an event listener that fails when we're rebasing
+								// the changes that move from one schema to another, because we try to go from an optional rootField
+								// to a forbidden one.
+								// assertSchema(peer, schemaGeneralized);
```

Comment:
Can we make it so the schema change is outside of the range of changes/commits that get rebased, so we can call `assertSchema()` like the other tests?